pkg/apis/kops/v1alpha2/cluster.go

```diff
@@ -349,12 +349,21 @@ type ClusterSubnetSpec struct {
 	// ProviderID is the cloud provider id for the objects associated with the zone (the subnet on AWS)
 	ProviderID string `json:"id,omitempty"`
 
-	// Egress defines the method of traffic egress for this subnet
-	Egress string `json:"egress,omitempty"`
+	// Egress is used to add more routes of precreated NAT gateways or NAT instances or VPC peering connections
+	Egress []EgressSpec `json:"egress,omitempty"`
```

Comment:
Technically I don't think we can make this change (unless you've done some magic somewhere), because it is incompatible with existing clusters that specify a string.  We could create v1alpha3 (which is why it would be nice to introduce the Networking object at the same time), or we could add a new field e.g. `Egresses` in the existing version.